model/flick.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+//Schema
+const flickSchema = Schema({
+  name: { type: String, required: true },
+  desc: { type: String, required: true },
+  userID: { type: Schema.Types.ObjectId, required: true },//make sure its part of the correct gallery
+  galleryID: { type: Schema.Types.ObjectId, required: true },
+  imageURI: { type: String, required: true, unique: true },//what comes back from s3
+  objectKey: { type: String, required: true, unique: true },//also comes back from s3
```

Comment:
I like that you are taking notes to remind yourself what things are.  I take notes in my code as well.  Sometimes if I have questions I will note them in there as well and just remove them before pushing.